{
  "cve": {
    "data_type": "CVE",
    "data_format": "MITRE",
    "data_version": "4.0",
    "CVE_data_meta": {
      "ID": "CVE-2018-14361",
      "ASSIGNER": "cve@mitre.org"
    },
    "affects": {
      "vendor": {
        "vendor_data": [

        ]
      }
    },
    "problemtype": {
      "problemtype_data": [
        {
          "description": [

          ]
        }
      ]
    },
    "references": {
      "reference_data": [
        {
          "url": "https://github.com/neomutt/neomutt/commit/9e927affe3a021175f354af5fa01d22657c20585",
          "name": "https://github.com/neomutt/neomutt/commit/9e927affe3a021175f354af5fa01d22657c20585",
          "refsource": "MISC",
          "tags": [

          ]
        },
        {
          "url": "https://lists.debian.org/debian-lts-announce/2018/08/msg00001.html",
          "name": "[debian-lts-announce] 20180802 [SECURITY] [DLA 1455-1] mutt security update",
          "refsource": "MLIST",
          "tags": [

          ]
        },
        {
          "url": "https://neomutt.org/2018/07/16/release",
          "name": "https://neomutt.org/2018/07/16/release",
          "refsource": "MISC",
          "tags": [

          ]
        }
      ]
    },
    "description": {
      "description_data": [
        {
          "lang": "en",
          "value": "An issue was discovered in NeoMutt before 2018-07-16. nntp.c proceeds even if memory allocation fails for messages data."
        }
      ]
    }
  },
  "configurations": {
    "CVE_data_version": "4.0",
    "nodes": [

    ]
  },
  "impact": {
  },
  "publishedDate": "2018-07-17T17:29Z",
  "lastModifiedDate": "2018-08-04T01:29Z"
}